src/components/common/NameRequestInfo.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <div id="name-request-info">
+    <!-- display name request info -->
```

Comment:
When code-folded, this component's template looks like this:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/b97493d4-c026-468c-a683-ce431a243858)